src/services/document-service.spec.js

```diff
@@ -312,6 +313,35 @@ describe('document-service', () => {
 
   });
 
+  describe('getAllExportableDocumentsMetadata', () => {
+
+    const testDocs = [
+      { title: 'doc-1', archived: false, origin: DOCUMENT_ORIGIN.internal },
+      { title: 'doc-2', archived: true, origin: DOCUMENT_ORIGIN.internal },
+      { title: 'doc-3', archived: false, origin: `${DOCUMENT_ORIGIN.external}/some-site` }
+    ];
+
+    let result;
+
+    beforeEach(async () => {
+      await Promise.all(testDocs.map(testDoc => createTestDocument(container, user, testDoc)));
+      result = await sut.getAllExportableDocumentsMetadata();
+    });
+
+    it('should return internal unarchived documents', () => {
+      expect(result.find(doc => doc.title === 'doc-1')).not.toBeUndefined();
```

Comment:
There's `toBeDefined` for this case

Reply:
I was also confused by this line :)

Reply:
I wanted you to be not unconfused! Nevrtheless, I changed it, thanks :-)

Reply:
That I for one am most surely not sure if I am anymore.

Reply:
:exploding_head: